app/models/reducer.rb

```diff
@@ -106,6 +107,19 @@ def reduce_into(extracts, reduction)
     raise NotImplementedError
   end
 
+  def extract_fetcher
```

Comment:
Not blocking: it feels like this could be extracted out of the reducer to the call site vs having the reducer know about how the fetchers are configured.

Reply:
Counterargument: the call site would have to look up `running_reduction?`, and `topic` from the reducer, and then do something with it.  It's a choice between two different code smells I think.